src/main/java/com/softserve/academy/spaced/repetition/service/DeckService.java

```diff
@@ -104,8 +112,21 @@ public void createNewDeck(Deck newDeck, Long category_id) throws NotAuthorisedUs
         deck.setDescription(newDeck.getDescription());
         deck.setCategory(categoryRepository.findById(category_id));
         deck.setDeckOwner(user);
-        Deck save = deckRepository.save(deck);
-        newDeck.setId(save.getId());
+        Deck savedDeck = deckRepository.save(deck);
+        newDeck.setId(savedDeck.getId());
+    }
+
+    @Transactional
+    public Deck createNewDeckAdmin(Deck newDeck) throws NotAuthorisedUserException {
```

Comment:
You have to add this deck to admin's cabinet also, haven't you?
And to delete it from there as well then.

Reply:
yes it is true. admin is a user.

Reply:
We need to decide what to do with the folder and after that I will correct it

Reply:
What's wrong with the folder?

Reply:
The big loop - get all forders users

Reply:
Sorry, I still cannot understand you. Why can't you just add them to the only user's folder?

Reply:
When I will delete from folder I will use mathod with big loop. Andriy told about it. So I'll use it. ok.

Reply:
I've understood but I don't know what to do. In any case we're likely to forget to implement it later.